### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by vegasmorph**
pr to add Terra Tritium ( $TRIT )

### CODE REVIEW

File: terra/assetlist.json [hunk 1]
```diff
@@ -5688,6 +5688,40 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/neb.png"
+        },
+         {
```
Comment (JeremyParish69) [inline terra/assetlist.json:5692]: This appears to have deviated from the schema. Please repair

File: terra/assetlist.json [hunk 2]
```diff
@@ -5688,6 +5688,40 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "TRIT",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra1g6fm3yu79gv0rc8067n2nnfpf0vks6n0wpzaf4u7w48tdrmj98zsy7uu00",
+      "name": "Terra Tritium Token",
+      "display": "Terra Tritium",
```
Comment (JeremyParish69) [inline terra/assetlist.json:5708]: This unit is not defined

File: terra/assetlist.json [hunk 3]
```diff
@@ -5688,6 +5688,40 @@
...
+      "symbol": "TRIT",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/terra-tritium/assets/main/trit.png"
```
Comment (JeremyParish69) [inline terra/assetlist.json:5711]: Please upload the logo image to this registry

File: terra/assetlist.json [hunk 4]
```diff
@@ -5688,6 +5688,40 @@
...
+      },
+      "coingecko_id": "Terra-Tritium",
```
Comment (JeremyParish69) [inline terra/assetlist.json:5713]: This is not a valid cgid

**Comment by JeremyParish69** [comment]
Looks like someone else added Tritium, so if all looks good on that one, we can close this PR?
https://github.com/cosmos/chain-registry/pull/3536

**Comment by vegasmorph** [comment]
yes of course . tks mate.

**Pull request closed without merging**